Tribler/Core/Modules/restapi/util.py

```diff
@@ -26,6 +28,38 @@ def return_handled_exception(request, exception):
     })
 
 
+def convert_channel_metadata_to_tuple(metadata):
+    """
+    Convert some given channel metadata to a tuple, similar to returned channels from the database.
+    :param metadata: The metadata to convert.
+    :return: A tuple with information about the torrent.
+    """
+    votes = 1
+    my_vote = 2
+    spam = 0
+    relevance = 0.9
```

Comment:
Could you add a todo here since this relevance score is not based on anything?

Reply:
Done